components/HeroVideo.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+
+interface HeroVideoProps {
+  categoryVideo: Boolean;
+}
+
+const HeroVideo: React.FC<HeroVideoProps> = ({categoryVideo = false}) => {
+  const [showAnimation, setShowAnimation] = useState(false);
+  const [phoneScreen, setPhoneScreen] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+    //   setWindowWidth(window.innerWidth);
+      if (window.innerWidth < 640) {
+        setPhoneScreen(true);
+        // document.body.style.overflow = "auto";
```

Comment:
Unless they are explaining how the code works, please remove the commented out lines